index.js

```diff
@@ -119,9 +125,28 @@ luqs.station = (kuerzel, options = {}) => {
           steckbrief.start_messung,
           steckbrief.ende_messung
         ] = tmpSteckbrief
+
         steckbrief.image = `${messortBildUrl}${steckbrief.kuerzel.toUpperCase()}.jpg`
         steckbrief.longitude = steckbrief.longitude.replace(',', '.')
         steckbrief.latitude = steckbrief.latitude.replace(',', '.')
+
+        if (options.format === 'geojson') {
+          geojson.features.push({
+            type: 'Feature',
+            geometry: {
+              type: 'Point',
+              coordinates: [
+                parseFloat(steckbrief.longitude),
+                parseFloat(steckbrief.latitude)
+              ]
+            },
+            properties: {
+              ...steckbrief
+            }
+          })
+          resolve(geojson)
```

Comment:
not returning here would call `resolve` twice.

```suggestion
          resolve(geojson)
          return
```